src/Factors.hpp

```diff
@@ -39,6 +40,11 @@ namespace motors_roboteq_canopen {
          */
         double max_current = 1000;
 
+        /** Factor to multiply the absolute encoder position value */
+        float encoder_position_factor = 1.0;
+
+        float absoluteEncoderPositionToSI(int32_t position) const;
```

Comment:
`absoluteEncoderPosition` is ambiguous. An incremental encoder (what we have) gives out a position that is absolute w.r.t. an arbitrary zero (i.e. absolute with an offset). We generally do not call that "absolute position".

I think that encoderPositionToSI would be enough (or even encoderToSI)